### ISSUE TITLE
Mapping Hue Smart Button via zha_event

### ISSUE THREAD

**Issue by Henhart** [azogue/eventsensor#47]
Hi, 

how can I find out the correct mapping for my hue smart button?

{
    "event_type": "zha_event",
    "data": {
        "device_ieee": "zzz",
        "unique_id": "yyy",
        "device_id": "xxx",
        "endpoint_id": 1,
        "cluster_id": 6,
        "command": "on",
        "args": []
    },
    "origin": "LOCAL",
    "time_fired": "2022-02-11T21:48:24.265319+00:00",
    "context": {
        "id": "aaa",
        "parent_id": null,
        "user_id": null
    }
}

My configuration:
name: smart_button_1
event name: zha_event
state: command
unique_id: yyy
Mapping: ?

Currently there is only the status "on" and "off_with_effect" working.

Thanks.

**Comment by oneseventhree** [comment]
Did you ever find out? I'd love to know!

**Comment by Henhart** [comment]
No, sorry. Unfortunately I still have no solution. Would anyone have a solution for this issue? Thanks.
